json-schema/schema.json

```diff
@@ -193,11 +198,11 @@
     "require_any_field": {
       "$comment": "Please list all fields here so that we can force the existence of one of them in other parts of the schemas.",
       "anyOf": [
-        {"required": ["processing:expression"]},
-        {"required": ["processing:lineage"]},
-        {"required": ["processing:level"]},
-        {"required": ["processing:facility"]},
-        {"required": ["processing:software"]}
+        {"type": "object", "required": ["processing:expression"]},
```

Comment:
This can easily be enforced in the object that also contains the anyOf  so that we at least don't need to repeat these 4 times.